src/storage/src/hummock/compactor/compactor_runner.rs

```diff
@@ -59,9 +54,8 @@ impl CompactorRunner {
         let compactor = Compactor::new(
             context.context.clone(),
             options,
-            context.sstable_store.clone(),
             key_range,
-            cache_policy,
+            CachePolicy::NotFill,
```

Comment:
Is there some background about why we use `CachePolicy::NotFill` now ?

Reply:
Because it is not necessary for compactor to fill cache.